syntax = "proto3";

package map_pb;

import "svg.proto";

message OutputSettings {
    double width                     = 1;
    double height                    = 2;
    double padding                   = 3;
    double line_width                = 4;
    double stop_radius               = 5;

    int32  bus_label_font_size       = 6;
    Point  bus_label_offset          = 7;

    int32  stop_label_font_size      = 8;
    Point  stop_label_offset         = 9;

    ColorVar underlayer_color        = 10;
    double   underlayer_width        = 11;

    repeated ColorVar color_palette  = 12;
}